src/tidy.zig

```diff
@@ -147,7 +147,6 @@ const DeadDetector = struct {
 
     fn is_entry_point(file: FileName) bool {
         const entry_points: []const []const u8 = &.{
-            "benchmark.zig",
```

Comment:
Funny how this tidy check was explicitly designed to catch dead files like this, but this particular one dodged the check, because I was lazy and implemnted the check based solely on the file name, and not on the entire path (and we used to have benchmark entry points)